### PR TITLE
GOK-92 | add. Inventory Listing screen to display stocks with its quantity

### PR DESCRIPTION
**Pull request by abinaya-u**
## Requirements

- [x] This PR has a proper title that briefly describes the work done
- [x] I have squashed / amended the comments to make it more redable
- [x] I have included link to all the JIRA ticket('s)
- [x] I wrote the code as a pair or atleast performed a self-review of my own code
- [x] I have updated the documentation on [Bahmni Wiki](https://bahmni.atlassian.net/wiki/spaces/BAH/overview) or [README](https://github.com/Bahmni/bahmni-lab-frontend/blob/main/README.md) (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Summary
This PR contains changes to show the stock items along with their quantities

## Screenshots
image

### CODE REVIEW

**Comment by gsluthra** [comment]
The Quantity column (number) should be "right aligned" for easy reading of numbers. Currently in screenshot it is left aligned. https://uxpickle.com/ux-best-practice-for-data-table-cell-content-alignment/

File: constants.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+export const inventoryMenu = ['Inventory', 'Dispense']
+export const locationCookieName = "bahmni.user.location"
+export const headers = [
+  {
+    key: "productName",
+    header: "Product Name",
+  },
+  {
+    key: "actualQuantity",
+    header: "Actual Quantity",
```
Comment (gsluthra) [inline constants.js:12]: Should be called "Current Quantity"

File: webpack.config.js [hunk 2]
```diff
@@ -1,79 +1,98 @@
...
+		 * refresh as a fallback in case of build failures
+		 */
+		hot: true,
+		/** "liveReload"
+		 * disable live reload on the browser. "hot" must be set to false for this to work
+		 */
+		liveReload: true,
+		proxy: {
+            '/openmrs': {
+              target: 'http://localhost:8080', // Replace with your backend URL and port
```
Comment (gsluthra) [inline webpack.config.js:57]: Seems hardcoded to localhost?

File: webpack.config.js [hunk 3]
```diff
@@ -1,79 +1,98 @@
...
+	/** "target"
+	 * setting "node" as target app (server side), and setting it as "web" is
+	 * for browser (client side). Default is "web"
+	 */
+	target: "web",
+	devServer: {
+		/** "port"
+		 * port of dev server
+		 */
+		port: "9500",
```
Comment (gsluthra) [inline webpack.config.js:36]: Why is this port hardcoded here?

File: webpack.config.js (hunk 2 above)
Comment (abinaya-u) [reply webpack.config.js:57]: proxy rule is added for  webpack-dev-server.. only for development purpose

File: webpack.config.js (hunk 3 above)
Comment (abinaya-u) [reply webpack.config.js:36]: This port again is hard coded for local development

File: constants.js (hunk 1 above)
Comment (abinaya-u) [reply constants.js:12]: We went with the table header name given in the mock up. If required we can change this

File: webpack.config.js [hunk 4]
```diff
@@ -1,79 +1,98 @@
...
+		 * being searched"
+		 */
+		rules: [
+			{
+				test: /\.(js|jsx)$/, //kind of file extension this rule should look for and apply in test
+				exclude: /node_modules/, //folder to be excluded
+				use: "babel-loader", //loader which we are going to use
+			},
+			{
+                test: /\.s[ac]ss$/i,
```
Comment (sikarvarsunil) [inline webpack.config.js:86]: @abinaya-u Please apply indentation

**Review by sikarvarsunil — APPROVED** [review]
Please apply indentation in all files

**Review by sikarvarsunil — COMMENTED** [review]
Please apply indentation in all files

**Review by pankaj2961 — APPROVED** [review]
LGTM

File: constants.js (hunk 1 above)
Comment (gsluthra) [reply constants.js:12]: Actual Qty doesn't make sense. It should be either: `Available Quantity` or `Current Quantity` or `Quantity In Stock` or `Quantity in hand`.

**Pull request merged**